config/default

```diff
@@ -11,8 +11,12 @@ assets = os.path.join(media.mypath, "assets")
 
 if not os.path.exists(assets):
   print "Extracting assets..."
-  with tarfile.open("{}.tbz2".format(assets), "r:*") as tf:
-    tf.extractall(media.mypath)
+  try:
+    with tarfile.open("{}.tbz2".format(assets), "r:*") as tf:
+      tf.extractall(media.mypath)
+  except tarfile.ReadError, e:
```

Comment:
How about this?
except (tarfile.ReadError, IOError), e:

Reply:
if the file doesn't exists, we should also have a reminder

Reply:
This patch is only meant to address the most common case that users may encounter.